notebooks/dynamicrafter-animating-images/dynamicrafter-animating-images.ipynb

```diff
@@ -0,0 +1,696 @@
+{
```

Comment:
<p>I'm not sure that description is relevant for us, we are mostly interested how model works on inference stage than training process</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>The description of the training was removed</p>